HST/pipeline/pipeline_update_hst_visit.py

```
@@ -81,9 +80,8 @@
 
 try:
     update_hst_visit(formatted_proposal_id, visit, logger)
-except:
-    # Before raising the error, remove the task queue of the proposal id from database.
-    remove_all_tasks_for_a_prog_id(formatted_proposal_id)
+except Exception as e:
+    logger.error(e)
```

Ruff (0.15.13)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)